core/build.gradle.kts

```diff
@@ -128,7 +128,10 @@ val samplesTest = tasks.register<Test>("samplesTest") {
     ignoreFailures = true
 
     testClassesDirs = fileTree("${layout.buildDirectory.get().asFile.path}/classes/testWithOutputs/kotlin")
-    classpath = files("${layout.buildDirectory.get().asFile.path}/classes/testWithOutputs/kotlin") + configurations["samplesRuntimeClasspath"] + sourceSets["main"].runtimeClasspath
+    classpath =
```

Comment:
Could you please add any comment, why classpath consist from these three parts

Reply:
I'm not 100% sure actually, I just reformatted the line because it was too long